### ISSUE TITLE
[Deprecation] URL.createObjectURL with media streams is deprecated and will be removed in M71, around December 2018.

### ISSUE THREAD

**Issue by feedmeastraycat** [sinch/sinch-js-rtc#21]
Error message in the browser. Will there be a fix for this before it goes away? :) 

```
[Deprecation] URL.createObjectURL with media streams is deprecated and will be removed in M71, around December 2018. Please use HTMLMediaElement.srcObject instead. See https://www.chromestatus.com/features/5618491470118912 for more details.
```

**Comment by k-grant** [comment]
+1 on this. We use this currently in production and it would be bad if things just stopped working.

**Comment by feedmeastraycat** [comment]
@k-grant Just as an FYI, the SDK version available on Sinch page is 1.4.11: 

https://www.sinch.com/downloads/

1.4.11 does not have this error. So I'm closing this issue.

I've been in contact with their developer support email and I told them about this, the fact that this repository hasnt been updated. The one responding said he was gonna look into it...

Ive been having a lot of issues with Sinch on the web the last couple of weeks so I might be forced to move over to Twilio instead. :(
